src/include/cursor.h

```diff
@@ -54,6 +54,7 @@ struct __wt_cursor_backup {
 
     WT_CURSOR *incr_cursor; /* File cursor */
 
+    bool has_ckpt;        /* Object has been checkpointed */
     bool incr_init;       /* Cursor traversal initialized */
```

Comment:
I think having multiple boolean fields means we should think about making these bit flags instead.

Reply:
I don't feel strongly -- this isn't memory that gets allocated a lot.

Reply:
Yep - 🤷 two is my limit of bool fields like this. I could go either way.

Reply:
I moved the boolean fields to flag values.